### DISCUSSION TITLE
怎么获取群成员的qq等级，或者好友的qq等级

### DISCUSSION THREAD

**Discussion by fhh-coder** [simple-robot/Discussions#80] [Q&A] [answered]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

v3怎么获取qq等级呢？比如某个群成员，或者qq好友？

**Comment by ForteScarlet** [comment, marked as answer]
参考 https://github.com/simple-robot/simbot-component-mirai/issues/116
